README.md

```diff
@@ -133,6 +133,7 @@ Go the the right directory and install dependencies:
 ```bash
 cd eslint-config
 npm install
+npx husky install
```

Comment:
hm is this really needed if it is part of the dev dependencies?

Reply:
As far as I understand the Readme this section is about your local environment/setup, since "[Why husky doesn't autoinstall anymore](https://blog.typicode.com/husky-git-hooks-autoinstall/)" is now a thing we need to add this step somewhere if we want to have the git hooks enabled as well.

Reply:
ahh okay 👍